packages/block-editor/src/components/media-placeholder/README.md

```diff
@@ -65,6 +65,24 @@ Class name added to the placeholder.
 - Type: `String`
 - Required: No
 
+### disableDropZone
+
+If true, the Drop Zone will not be rendered. Users won't be able to drag & drop any media into the component or the block. The UI controls to upload the media via file, url or the media library would be intact.
+
+- Type: `Boolean`
+- Required: No
+- Default: `false`
+
+### dropZoneUIOnly
+
+If true, only the Drop Zone will be rendered. No UI controls to upload the media will be shown.
+This prop will take precedence over `disableDropZone`.
+It means, if both the props are supplied with `true` value, the component will still render the Drop Zone.
```

Comment:
I don't think this line is completely true now, when both props are `true` nothing will be displayed. 

I think that logically makes sense. Could change this to say something like 'The `disableDropZone` prop still takes precedence over `dropZoneUIOnly`—specifying both as true will result in nothing to be rendered.`

Reply:
ee1b23822b215b587289570dbd57112f11c14355